style.css

```diff
@@ -515,3 +523,365 @@ input[type="text"] {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
 }
+
+@media screen and (min-width: 768px) {
+  .main-header {
```

Comment:
As your portfolio has to be responsive.
- kindly fix the width of your background image to fit on the screen while you increment the resolution of your screen.
- Maintain your header with the nav bar responsive too
- 

Here's an example where the elements are not aligned as in the figma template

![image](https://user-images.githubusercontent.com/93630700/229522951-943fde23-2ff5-4ae0-9508-91b86b09e002.png)